apps/design-land/src/app/image/image.component.html

```diff
@@ -1 +1,28 @@
-<daff-image src="/assets/card/uber-logo.svg" alt="uber logo" width="300" height="119"></daff-image>
\ No newline at end of file
+<daff-article>
+  <h1 daffArticleTitle>Image</h1>
+  <p daffArticleLead>
+    The image component is used to customize an image you want to display in your project. To those familiar with 
+    <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img" target="_blank">the HTML <code>img</code></a>, 
+    <code>daff-image</code> should feel similar. <code>daff-image</code> is an opinionated version that encourages end-user friendly usage.
+  </p>
+
+  <h3>Usage</h3>
+  <p>
+    You can modify the <code>width</code>, <code>height</code>, <code>alt</code>, and <code>src</code> of the image component.
+  </p>
+
+  <h3>Sizes</h3>
+  <p>
+    The size of an image can be updated by using the <code>height</code> and <code>width</code> of the property. 
+    Any size is supported in pixels. Height and width are required to display the image.
+  </p>
+
+  <h3>Basic Image</h3>
+  <design-land-example-viewer-container example="basic-image"></design-land-example-viewer-container>
+
+  <h3>Image Load Output</h3>
+  <design-land-example-viewer-container example="load-image"></design-land-example-viewer-container>
+
+  <h3>Accessibility</h3>
+  <p>Daffodil uses native <code>alt</code> to ensure an accessible experience by default.</p>
```

Comment:
`Images should be given a meaningful description using the native <code>alt</code> attribute to ensure an accessible experience by default. An error will be thrown if `<golf-image>` is missing an <code>alt</code> attribute.`